src/styles/index.scss

```diff
@@ -58,30 +66,29 @@ width: 100%;
 	/* box-shadow: 0 0 0 0.2rem rgba(207, 23, 23, 0.25); */
 }
 .form-group {
-
 	width: 100%;
 	padding: 3px;
 }
 
-.form-label{
-display: flex;
-justify-content: center;
-margin: 0;
+.form-label {
+	display: flex;
+	justify-content: center;
+	margin: 0;
 }
 
-:global(.form-group){
+:global(.form-group) {
 	margin-bottom: 0;
 }
 
-.pagination-part{
+.pagination-part {
 	display: flex;
 	justify-content: center;
 	padding-top: 8px;
 }
 
-
-
-.page-link, .page-item:first-child .page-link ,.page-item:last-child .page-link{
+.page-link,
+.page-item:first-child .page-link,
+.page-item:last-child .page-link {
```

Comment:
What's the purpose of this? Can't it just be `.page-link`?

Reply:
i targeted the class when i was styling some paginations